File: PythonAssignment/JavaAssignment/Chapter 3 Tasks/question3.java
public class question3 {
    public static void main(String[] args) {
        
        for (int row = 0; row < 10; row++) {
                       for (int column = 0; column < 10; column++) {
                
                System.out.print(row % 2 == 1 ? "<" : ">");
            }
        }
       
        System.out.println();
    }
}

File: PythonAssignment/PythonAssignment/Chapter 3 Tasks/question3.py
for row in range(10):
	for column in range(10):
		print('<' if row % 2 == 1 else '>', end='')
print()

#When row is an even number it prints '>' 10 times
#When row is an odd number it prints '<' 10 times
#end='' makes the column to print on the same line.
#the print() is not indented, :: it does'nt have an efect in the code
#if indented then after column is done it moves to the next line!